src/renderers/WebGLRenderer.js

```diff
@@ -1147,6 +1151,61 @@ THREE.WebGLRenderer = function ( parameters ) {
 
 	}
 
+	this.renderOverride = function ( overrideMaterial, scene, camera, renderTarget, clearColor, clearAlpha ) {
+
+		var originalClearColor = this.getClearColor(), originalClearAlpha = this.getClearAlpha(), originalAutoClear = this.autoClear;
+
+		this.autoClear = false;
```

Comment:
I do like the defaults, since they are actually meaningful, user has to think about them and may get them wrong. Just think that special case should be considered too.